theano/tensor/tests/test_opt.py

```diff
@@ -4050,7 +4050,7 @@ def test_local_useless_tile(self):
                 f(data)
                 
                 # Check that stacktrace is copied over
-                self.assertTrue(hasattr(f.outputs[0].variable.tag, 'trace'))
+                self.assertTrue(check_stack_trace(f, ops_to_check='all'))
                 self.assertTrue(len(f.outputs[0].variable.tag.trace)>0)
```

Comment:
I think this should be `self.assertTrue(len(f.maker.fgraph.outputs[0].tag.trace)>0)` to check the length of the trace in **optimized** function, but that doesn't pass because the trace is empty. @lamblin